scripts/saveCanvas.js

```diff
@@ -0,0 +1,24 @@
+/**
+  Fetch the elements within the canvas then, using _serializeCanvas,
+  create a string represneting that data. Store the resulting string
+  at localStorage.set(storageName).
+
+   @param {string} storageName - a key for a value in localStorage
```

Comment:
These functions save the canvas to local storage and then restore the canvas afterwards.